src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -385,4 +394,9 @@
     <AdditionalFiles Include="PublicAPI.Shipped.txt" />
     <AdditionalFiles Include="PublicAPI.Unshipped.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <None Update="ProjectSystem\Rules\Items\EditorConfigFiles.BrowseObject.xaml">
```

Comment:
You can remove this, its CPS being over-zealous with the project file

Reply:
Done.